bundles/org.eclipse.equinox.preferences/build.properties

```diff
@@ -19,7 +19,8 @@ bin.includes = META-INF/,\
                plugin.properties,\
                .options,\
                about.html,\
-               about_files/
+               about_files/,\
+               OSGI-INF/org.eclipse.core.internal.preferences.BundleStateScopeServiceFactory.xml
```

Comment:
I'm sure you already noticed, but just in case, this is what is causing the PR build to fail.

Reply:
Done.